src/components/Filter/filter.jsx

```diff
@@ -0,0 +1,54 @@
+import {useState} from "react"
+
+const Filter = ({ search }) => {
+  const [ displayDropdown, setDisplayDropdown ] = useState(false);
```

Comment:
If this is a function that is coming from outside, call it something like `onSearch` so we will know that this is a function and not a variable